.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
FYI, if you want to build v. tiny images, you can use something like this: https://github.com/clux/muslrust

Reply:
this isn't the image building ! it's the github action workflow config that runs my CI checks. this is the actual Dockerfile that uses muslrust.